sg_jira/bridge.py

```diff
@@ -23,6 +24,12 @@
 # Ensure basic logging is always enabled
 logging.basicConfig(format="%(levelname)s:%(name)s:%(message)s")
 
+# The bridge webserver is multithreaded, which means we need to
+# track Shotgun connections via the API per thread. The SG Python
+# API is not threadsafe, and using a single, global connection
+# across all threads will lead to some weird behavior.
+_g_sg_cached_connections = threading.local()
```

Comment:
Might as well make this a "private" class attributes, as it doesn't need to be accessible from anywhere else. What do you think?